### ISSUE TITLE
update to remove keep-terminated-pod-volumes (ocp)

### ISSUE THREAD

**Issue by faust64** [aquasecurity/kube-bench#405]
As a follow-up for kube-bench/pull/162

I did notice that the keep-terminated-pod-volumes check was indeed removed from k8s 1.11 tests. Although it was not from the ocp-3.11.
Running `kube-bench node --version ocp-3.11` does turn with a fail on 7.9, which is no longer relevant.

Thanks!

**Comment by lizrice** [comment]
The OCP tests are based on the Red Hat Hardening Guide. I'll check in with them case there is an update to that.

**Comment by faust64** [comment]
right ... we're still waiting on a v1.3. While the 1.2 is indeed mistakenly still mentioning this.
Disregard for now. Thanks for your follow-up.
